<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>563</width>
				<height>641</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<property name="styleSheet">
			<string notr="true">background-color: rgb(255, 161, 67);</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QLabel" name="label">
				<property name="geometry">
					<rect>
						<x>150</x>
						<y>20</y>
						<width>271</width>
						<height>61</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>Arial</family>
						<pointsize>20</pointsize>
						<underline>false</underline>
						<kerning>true</kerning>
					</font>
				</property>
				<property name="text">
					<string>Cadastro de produtos</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_2">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>160</y>
						<width>111</width>
						<height>31</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>15</pointsize>
					</font>
				</property>
				<property name="text">
					<string>Código</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_3">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>250</y>
						<width>111</width>
						<height>31</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>15</pointsize>
					</font>
				</property>
				<property name="text">
					<string>Descrição</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_4">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>350</y>
						<width>111</width>
						<height>31</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>15</pointsize>
					</font>
				</property>
				<property name="text">
					<string>Valor</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_10">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>400</y>
						<width>111</width>
						<height>31</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<pointsize>15</pointsize>
					</font>
				</property>
				<property name="text">
					<string>Quantidade</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_5">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>230</y>
						<width>291</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>Descrição breve. Ex: Camiseta, Calça, Jaqueta, etc...</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_6">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>330</y>
						<width>291</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>Inserir o valor no formato inglês. Ex: R$ 100,00 -&gt; 100.00.</string>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>450</y>
						<width>171</width>
						<height>41</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(255, 186, 125);</string>
				</property>
				<property name="text">
					<string>Enviar</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_7">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>140</y>
						<width>291</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>Inserir apenas números.</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_codigo">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>159</y>
						<width>311</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(255, 255, 255);</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_descricao">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>250</y>
						<width>311</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(255, 255, 255);</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_valor">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>350</y>
						<width>311</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(255, 255, 255);</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_quantidade">
				<property name="geometry">
					<rect>
						<x>200</x>
						<y>400</y>
						<width>311</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">background-color: rgb(255, 255, 255);</string>
				</property>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>563</width>
					<height>21</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
